machine_learning/scoring_functions.py

```diff
@@ -16,6 +16,16 @@
 
 # Mean Absolute Error
 def mae(predict, actual):
+    """
+        Examples(rounded for precision):
+        >>> actual = [1,2,3];predict = [1,4,3]
+        >>> np.around(mae(predict,actual),decimals = 2)
+        0.67
+
+        >>> actual = [1,1,1];predict = [1,1,1]
+        >>> mae(predict,actual)
+        0.0
+    """
```

Comment:
Please dedent lines 20-27 so they left align with lines 19 and 28.  Repeat below.